angular/base/tslint.json

```diff
@@ -13,7 +13,8 @@
     "deprecation": {
       "severity": "warning"
     },
-    "component-class-suffix": true,
+    // Ionic CLI require to allow "Page"
```

Comment:
Could you just remove the comments in the json?

Reply:
Yes, I did!